tests/integration/charts/cluster_with_medusa_local.yaml

```diff
@@ -7,6 +7,8 @@ cassandra:
     size: 1
   ingress:
     enabled: false
+  loggingSidecar:
```

Comment:
enabled is the default, so why bother readding them?

Reply:
You're right, I'll remove these.